elements/bulbs-reading-list/lib/reading-list-article.js

```diff
@@ -106,8 +106,39 @@ export default class ReadingListArticle {
     this.element.dataset.loadStatus = 'loading';
   }
 
+  setupTaboola () {
+    let taboola = window._taboola || [];
```

Comment:
Yeah, it looks like they're persisting the array

```
window._taboola = window._taboola || [];
_taboola.push();
```

With this expression window._taboola is never assigned:
```js
let taboola = window._taboola || [];
```
At that point `window._taboola` could still be undefined.

```
I think this would work:
```
window._taboola = window._taboola || [];
let taboola = window._taboola;
```